Osiris/Hacks/Backtrack.h

```diff
@@ -0,0 +1,76 @@
+#pragma once
+
+#include <deque>
+#include <algorithm>
+#include "../Memory.h"
+#include "../Interfaces.h"
+#include "../SDK/UserCmd.h"
+#include "../SDK/FrameStage.h"
+
+namespace Backtrack {
+	void init() noexcept;
+	void update(FrameStage) noexcept;
+	void run(UserCmd*) noexcept;
+
+	struct Record {
+		Vector absOrigin;
+		Vector mins;
+		Vector maxs;
+		Vector head;
+		int flags;
+		float simulationTime;
+        int layerOrder[15];
+        int layerSequence[15];
+        float layerWeight[15];
+        float layerCycle[15];
+        matrix3x4 matrix[128];
+    };
+
+    struct Cvars {
+        Cvar* updateRate;
+        Cvar* maxUpdateRate;
+        Cvar* interp;
+        Cvar* interpRatio;
+        Cvar* minInterpRatio;
+        Cvar* maxInterpRatio;
+        Cvar* maxUnlag;
+    };
+
+    extern Cvars cvars;
+
+    constexpr auto getLerp() noexcept
+    {
+        auto ratio = cvars.interpRatio->getFloat();
+        ratio = std::clamp(ratio, (cvars.minInterpRatio->getFloat() != 1.f) ? cvars.minInterpRatio->getFloat() : 1.f, cvars.maxInterpRatio->getFloat());
```

Comment:
Should do the same but cleaner:
`auto ratio = std::clamp(cvars.interpRatio->getFloat(), cvars.minInterpRatio->getFloat(), cvars.maxInterpRatio->getFloat());`